src/actions/actions.factory.js

```diff
@@ -20,7 +20,7 @@ module.exports = (
   START,
   STOP
 ) => {
-  return (parsedCommand, mapUsernameToUserInfo, myUserInfo, events, zone) => {
```

Comment:
so now time includes the time-zone as well?

Reply:
Yes. There is not much point passing in the zone here from App.js because we are getting the timezone from the user command. That's why I removed it.